webpack.common.config.js

```diff
@@ -100,10 +86,34 @@ module.exports = function (outputFilename) {
             'views/forms/inputs/TextSelect',
             'views/forms/components/ReadModeBar',
             'views/forms/inputs/ListInput',
-            'views/forms/inputs/SimpleCheckBoxSet'
+            'views/forms/inputs/SimpleCheckBoxSet',
+            'views/forms/inputs/GroupSelect',
+            'views/forms/inputs/BaseSearchableSelect',
+            'views/forms/inputs/MultiSearchableSelect',
+            'views/forms/inputs/SearchableSelect',
+            'views/Backbone.TableView',
+            'views/datalist/datalistBundle',
+            'views/tabs/tabsBundle',
+            'views/uploader/uploaderBundle',
+            'views/components/BaseFormDialog',
+            'views/components/BaseModalDialog',
+            'views/components/BaseWizard',
+            'views/components/ConfirmationDialog',
+            'views/components/MultiViewModalDialog',
+            'views/components/DropDown',
+            'views/wizard/BaseWizard',
           ].map(function (file) {
             return path.resolve(TARGET_JS, 'shared', file);
-          })
+          }).concat([
+            /moment-tz/,
+            /vendor\/plugins\/vkbeautify/,
```

Comment:
nit: do we need to escape the regular ```/``` ? Some below do not, and these ones here do ```\/```. Maybe we can have them all the same

Reply:
this is a regex literal, if it was a string we wouldn't have to

Reply:
doh.... missed that somehow
👍